tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -630,6 +630,39 @@ class WskBasicUsageTests
                 }
     }
 
+    it should "ensure --web flag does not remove existing annotations" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "webaction"
+            val file = Some(TestUtils.getTestActionFilename("echo.js"))
+            val annots = Map("someKey" -> JsString("someValue"))
+
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) => action.create(name, file, annotations = annots)
+
+            }
+
+            wsk.action.create(name, file, web = Some("true"), update = true)
+
+            val stdout = wsk.action.get(name, fieldFilter = Some("annotations")).stdout
+            assert(stdout.startsWith(s"ok: got action $name, displaying field annotations\n"))
+            removeCLIHeader(stdout).parseJson shouldBe JsArray(
+                JsObject(
+                    "key" -> JsString("web-export"),
+                    "value" -> JsBoolean(true)),
+                JsObject(
+                    "key" -> JsString("raw-http"),
+                    "value" -> JsBoolean(false)),
+                JsObject(
+                    "key" -> JsString("final"),
+                    "value" -> JsBoolean(true)),
+                JsObject(
+                    "key" -> JsString("someKey"),
+                    "value" -> JsString("someValue")),
```

Comment:
Suggest making "someKey" and "someValue" as constants for use in line 637 and here